src/Cohesity.Powershell/Scripts/Recover-CohesityBackupToView.ps1

```diff
@@ -0,0 +1,117 @@
+function Recover-CohesityBackupToView {
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false)]
+        [String]$SourceName,
+        [Parameter(Mandatory = $false)]
+        [String]$ViewName,
+        [Parameter(Mandatory = $false)]
+        [String]$QOSPolicy,
+        [Parameter(Mandatory = $false)]
+        [String]$ProtectionJobName,
+        [Parameter(Mandatory = $false)]
+        [String]$server,
+        [Parameter(Mandatory = $false)]
+        [String]$token
+    )
+    Begin {
+        if (-not (Test-Path -Path "$HOME/.cohesity")) {
+            throw "Failed to authenticate. Please connect to the Cohesity Cluster using 'Connect-CohesityCluster'"
+        }
+        $session = Get-Content -Path $HOME/.cohesity | ConvertFrom-Json
+
+        $server = $session.ClusterUri
+
+        $token = $session.Accesstoken.Accesstoken
+    }
+
+    Process {
+        $random = Get-Random -Minimum -100000 -Maximum 1000000
```

Comment:
Get timestamp instead of getting random numbers. Just like UI

Reply:
Fixed. using the current date as the random object for restore task name